charts/service/templates/01_rollout.yaml

```
@@ -79,6 +79,11 @@ spec:
             - primary
       {{- if $environment.rolloutAnalysis }}
       analysis:
+        {{- if $environment.rolloutAnalysis.custom }}
+        templates:
+        - templateName: {{ $environment.name }}
```

Shouldn't it be `$name` here? (the analysis template uses `$name` in the name)

```suggestion
        - templateName: {{ $name }}
```

---

Yep 🙈